src/dict.c

```diff
@@ -94,27 +103,25 @@ uint64_t dictGenCaseHashFunction(const unsigned char *buf, size_t len) {
 /* ----------------------------- API implementation ------------------------- */
 
 /* Reset hash table parameters already initialized with _dictInit()*/
-static void _dictReset(dict *d, int htidx)
-{
-    d->ht_table[htidx] = NULL;
-    d->ht_size_exp[htidx] = -1;
-    d->ht_used[htidx] = 0;
+static void _dictReset(dict *d) {
+    d->ht_table = NULL;
+    d->ht_size_exp = -1;
+    d->ht_size_exp_prev = -1;
+    d->ht_used = 0;
 }
 
 /* Create a new hash table */
-dict *dictCreate(dictType *type)
-{
+dict *dictCreate(dictType *type) {
     dict *d = zmalloc(sizeof(*d));
 
-    _dictInit(d,type);
+    _dictInit(d, type);
```

Comment:
Perhaps we shouldn't modify codes that are not related to this PR.

Reply:
Agree. The funny thing is that I run accidentally formatter. And later on manually undo (most of) the changes.